torch_geometric/loader/link_neighbor_loader.py

```diff
@@ -101,6 +101,9 @@ class LinkNeighborLoader(LinkLoader):
             replacement. (default: :obj:`False`)
         directed (bool, optional): If set to :obj:`False`, will include all
             edges between all sampled nodes. (default: :obj:`True`)
+        temporal_strategy (string, optional): The sampling strategy when using
```

Comment:
The PR title say "last_k", is this the same thing?

Perhaps we should add some explanation of the strategies, or link to other documentation (I guess they come from pyg_lib?)

Reply:
Good suggestion. Added more documentation.